api/handlers/job_test.go

```diff
@@ -81,5 +90,96 @@ var _ = Describe("Job", func() {
 				expectNotFoundError("Job")
 			})
 		})
+
+		Describe("org delete", func() {
+			var (
+				operation    string
+				resourceGUID string
+			)
+
+			BeforeEach(func() {
+				operation = "org.delete"
+				resourceGUID = "cf-org-guid"
+			})
+
+			When("org delete is in progress", func() {
+				BeforeEach(func() {
+					orgRepo.GetOrgReturns(repositories.OrgRecord{
+						GUID:      "cf-org-guid",
+						DeletedAt: time.Now().Format(time.RFC3339Nano),
+					}, nil)
+				})
+
+				It("returns a processing status", func() {
+					guid := operation + "~" + resourceGUID
```

Comment:
All that setup could be moved into `BeforeEach` - to initialise the request URL - and `JustBeforeEach` - to execute the request. Thus the setup repetition in `It`s would be removed and the test would be aligned with idiomatic Ginkgo style